tck/api/src/main/java/org/eclipse/microprofile/metrics/tck/MetricRegistryTest.java

```diff
@@ -106,9 +106,9 @@ public void useExistingMetaDataTest() {
     @Test
     @InSequence(5)
     public void testMetricRegistryType() {
-        Assert.assertEquals(MetricRegistry.Type.APPLICATION, metrics.getType());
-        Assert.assertEquals(MetricRegistry.Type.BASE, baseMetrics.getType());
-        Assert.assertEquals(MetricRegistry.Type.VENDOR, vendorMetrics.getType());
+        Assert.assertEquals(MetricRegistry.APPLICATION_SCOPE, metrics.getScope());
+        Assert.assertEquals(MetricRegistry.BASE_SCOPE, baseMetrics.getScope());
+        Assert.assertEquals(MetricRegistry.VENDOR_SCOPE, vendorMetrics.getScope());
```

Comment:
would be good to add another injected MetricRegistry with a custom scope, then test for that scope name here.